{"name": "test_one", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "self = <camp_study_interface.aek.testcase.test_login.TestLogin object at 0x0000022BE428B3D0>\n\n    def test_one(self):\n        print(\"hello\")\n        q=1\n        w=2\n>       assert  q==w\nE       assert 1 == 2\n\naek\\testcase\\test_login.py:28: AssertionError"}, "start": 1676207453203, "stop": 1676207453203, "uuid": "c592cd86-fd93-4d29-a07c-591a642f64b3", "historyId": "94523186c8599437c49f7504b2ea1ada", "testCaseId": "79c25fd06588700e2ee472b1e39a306d", "fullName": "aek.testcase.test_login.TestLogin#test_one", "labels": [{"name": "parentSuite", "value": "aek.testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "我叫小白菜"}, {"name": "thread", "value": "19324-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "aek.testcase.test_login"}]}